api/app/src/main/kotlin/packit/model/Packet.kt

```diff
@@ -1,25 +1,15 @@
 package packit.model
 
-import com.fasterxml.jackson.databind.ObjectMapper
 import jakarta.persistence.*
+import packit.helpers.JsonMapConverter
 
 @Entity
-@Table(name = "packet")
 data class Packet(
     @Id
     val id: String,
     val name: String,
     val displayName: String,
-    @Column(name = "parameters", columnDefinition = "json")
-    val parameters: String,
+    @Convert(converter = JsonMapConverter::class)
+    val parameters: Map<String, Any>,
```

Comment:
Do you mean `val parameters: Map<String, String>` instead of `val parameters: Map<String, Any>`

I can make the value string but `Any` data type makes it hard to fail.

Reply:
Yeah exactly, I seem to recall in Orderly 1 all parameters were actually strings, but not sure if that's still the case in Orderly 2. @hillalex ?